src/components/rich-text-editor/adapter/vue-froala.ts

```diff
@@ -0,0 +1,311 @@
+// Thus code is ugly but it is an adaptation of https://github.com/froala/vue-froala-wysiwyg to get rid of the global $ (jquery) dependency.
+// We should get rid of this code when https://github.com/froala/vue-froala-wysiwyg/issues/50 is resolved.
```

Comment:
When you read https://github.com/froala/wysiwyg-editor/issues/1668, it seems like you don't need this anymore.

Reply:
they removed the global $ usage from froala/wysiwyg-editor.  However, they haven't removed global $ usage from vue-froala yet (see https://github.com/froala/vue-froala-wysiwyg/blob/master/src/vue-froala.js, line #95).